PolyEngine/Engine/Src/Engine.cpp

```diff
@@ -27,6 +27,20 @@ void Poly::Engine::Deinit()
 void Engine::Update(float dt) {
 	Game->Update(dt);
 
+	// quite stupid test for input :P
+	for(int i = 0; i < InputBase.Size(); i++){
```

Comment:
while(InputBase.Size() > 0) whould be cleaner. No need to change it anyways as this is temporary code.